### ISSUE TITLE
Is hormone secretion intended only for endocrine, or includes other types?

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#17084]
[this issue is for not discussion of CHEBI roles. It pertains entirely to the usage of the term "hormone"]

We have:

```
id: GO:0046879
name: hormone secretion
namespace: biological_process
def: "The regulated release of hormones, substances with a specific regulatory effect on a particular organ or group of cells." [ISBN:0198506732]
is_a: GO:0008150 ! biological_process
is_a: GO:0009914 ! hormone transport
is_a: GO:0023061 ! signal release
```

There is only one subclass that indicates mode of dispersal:

```
id: GO:0060986
name: endocrine hormone secretion
namespace: biological_process
def: "The regulated release of a hormone into the circulatory system." 
is_a: GO:0046879 ! hormone secretion
relationship: part_of GO:0050886 ! endocrine process
```

There is no subclasses for autocrine, paracrine, etc.

I think GO:0060986 is fake precision. I think people read "hormone secretion" as being endocrine. 

If you query with GO:0060986 then you miss insulin secretion! There are only 87 mouse annotations to GO:0060986, as opposed to GO:0046879 for hormone signaling. From this we would conclude that endocrine hormone signaling was a rarer form of hormone signaling.

Also, not sure GO:0060986 def is correct. The def includes exocrine system. I think it intends to say releases *directly* into ducts.

Do we really need this level of precision at all? Mechanistically is there a difference in the gene product action between endocrine/exocrine/paracrine/autocrine?

If not, then we can simplify and merge

If there is, then we will have to do work to (a) properly clarify that the parent term encompasses different subtypes and include a do_not_annotate (since you always know the subtype) (b) introduce children not just for endocrine (c) reannotate so we do not have false precision and unnevenness

i haven't looked at the other usages of hormone

**Comment by cmungall** [comment]
I suppose the rationale for not putting insulin secretion under endocrine is the fact worms do not have a circulatory system, and we have worm annotations http://amigo.geneontology.org/amigo/reference/PMID:17289575

But this reinforces my point: do we even need this level of distinction?

**Comment by cmungall** [comment]
Aside:

We have nothing linking endocrine process to uberon or ncbitaxon hence to TCs, we have one worm annotation to an endocrine process (angiotensin maturation)

http://amigo.geneontology.org/amigo/gene_product/WB:WBGene00044150

This comes from an interpro2go:
http://www.ebi.ac.uk/interpro/entry/IPR033508

What needs fixed here? interpro2go or GO?

**Comment by addiehl** [comment]
From the wikipedia entry on hormones:
_Hormone secretion may occur in many tissues. Endocrine glands are the cardinal example, but specialized cells in various other organs also secrete hormones. Hormone secretion occurs in response to specific biochemical signals from a wide range of regulatory systems._ 

I think of hormones as acting at a distances across the body, but depending on the system, the action may occur more locally within a common tissue or organ.  Also, there is a bit of gray between hormones and some cytokines in terms of easy definition

**Comment by cmungall** [comment]
Exactly, any time there is a cardinal example of X, people will assume that X is an instance of the cardinal example and just use X, leaving the subclass cardinal-X untouched, leading to unevenness.

**Comment by ukemi** [comment]
I agree with @addiehl.  I remember discussing this with a group of physiologists and the difficulty we came to at the end of the discussion is what is meant by at a distance. As @addiehl says above it may occur quite locally. It's almost case by case based on historical usage.

**Comment by deustp01** [comment]
> id: GO:0046879
> name: hormone secretion
> namespace: biological_process
> def: "The regulated release of hormones, substances with a specific regulatory effect on a particular organ or group of cells." [ISBN:0198506732]

> id: GO:0060986
> name: endocrine hormone secretion
> namespace: biological_process
> def: "The regulated release of a hormone into the circulatory system." 

The problem may be at the GO end, with the second definition. Why does a hormone need to travel via a structure recognized by anatomists as a circulatory system in order to qualify as "endocrine"? Relax that constraint to something like "extracellular space" and then perhaps we can use additional criteria like "acting on cells other than the ones that secrete the hormone" or "capable of circulating" or "stable" to distinguish an endocrine one from a paracrine one (acts on the secreting cell or neighboring ones, often because the hormone is unstable (eicosanoids)) and an autocrine one (acts on the cell that produces it, for whatever reason). These distinctions suggest, correctly, that a hormone could have both paracrine and autocrine modes of action (eicosanoids again).

If we can generalize from the mammalian insulin and glucagon cases (no clue whether this is legitimate), an endocrine hormone also has the property that it does not act on the cell that produces it. If that is general, it is a really clean differentium.

**Comment by ukemi** [comment]
IIRC there were three differentia, endocrine, paracrine, autocrine.

**Comment by vanaukenk** [comment]
> we have one worm annotation to an endocrine process (angiotensin maturation)
> http://amigo.geneontology.org/amigo/gene_product/WB:WBGene00044150
> This comes from an interpro2go:  http://www.ebi.ac.uk/interpro/entry/IPR033508
> What needs fixed here? interpro2go or GO?

I think this InterPro mapping is too specific.  C. elegans does not have angiotensin. 

I've put in a ticket to fix this here:

https://github.com/geneontology/go-annotation/issues/2307

**Comment by addiehl** [comment]
Another consideration is that hormones are structurally diverse.  Some are polypeptides (proteins or smaller bits), others are small molecules.  Whereas cytokines are strictly proteins.

Basically, something is a hormone if it has been classically declared to be a hormone via some loose idea of what a hormone is, and others in the field have accepted this declaration.

**Comment by ValWood** [comment]
No action described here.

**Comment by pgaudet** [comment]
My understanding is that the action is to merge 'endocrine hormone secretion' into its parent 'hormone secretion'

**Comment by raymond91125** [comment]
There are no direct annotations with endocrine hormone secretion. It makes sense to flatten the hierarchy.

**Comment by raymond91125** [comment]
GO:0060986 endocrine hormone secretion is being used by
CL:0000163 endocrine cell in LD 'secretory cell' and ('capable of' some 'endocrine hormone secretion'
UBERON:0002368 endocrine gland SubClass Of 'capable of' some 'endocrine hormone secretion'

How does one go about resolving those? @pgaudet

**Comment by pgaudet** [comment]
Hi @raymond91125 
We need to open tickets in the [CL](https://github.com/obophenotype/cell-ontology) and the [UBERON](https://github.com/obophenotype/uberon) trackers 

Ideally the encocrine cells and endocrine glands should not be defined just as being able to secrete endocrine hormones, this seems quite a circular definition, isn't it? 

Also, is our definition of 'endocrine hormone secretion' "The regulated release of a hormone into the circulatory system." sufficiently different from 'hormone secretion', "The regulated release of hormones, substances with a specific regulatory effect on a particular organ or group of cells."?

**Comment by dosumis** [comment]
CL and Uberon need to retain a way to refer to endocrine hormone secretion as a way of defining elements of the endocrine system - which only includes cells secreting hormones into the circulatory system - not those only releasing them to act locally.  

I've proposed a pattern for this in a [comment on the CL ticket](https://github.com/obophenotype/cell-ontology/issues/1706#issuecomment-1327207879).  The idea is to extend GO process terms using  a 'has end target location'  relationship to blood or circulatory fluid (to include arthropods and at least some other invertebrates).   

To avoid nested class expressions in CL and Uberon, I'd prefer these extended process to be names terms in GO.  I think there is a good case for them purely for GO annotation purposes - as the release into the circulatory system (and regulation of that release) will have very different significance for whole organism physiology to local release.  In some cases the same molecular can have endocrine, paracrine and neurotransmitter roles depending on the context - see norepinephrine and serotonin. 

Question: would GO be willing not only to keep the endocrine secretion class, but to add subclasses for secretion of major endocrine hormones, following the suggested pattern above?  If so, we will use these to define endocrine organs and cell types in Uberon and CL.  If not, we will use nested class expressions.

**Comment by raymond91125** [comment]
How about
'hormone secretion' and ('has target end location' some 'circulatory system')?
Consistent with GO:0050886 'endocrine process' DEF 'The process that involves the secretion of or response to endocrine hormones. An endocrine hormone is a hormone released into the circulatory system.'

**Comment by raymond91125** [comment]
With regard to the original question, how do we specify target locations for autocrine and paracrine?

**Comment by dosumis** [comment]
> How about
> 'hormone secretion' and ('has target end location' some 'circulatory system')?

That was my first thought, but then I worried that it might apply to hormones acting in a paracrine fashion in the walls of blood vessels. 

> With regard to the original question, how do we specify target locations for autocrine and paracrine?  

Harder to do this in a parallel way.  Would need some way to define target as self or local to self.  COuld be defined by part_of relationships to paracrine cell-cell signalling and autocrine cell-cell signaling. 

For paracrine, I'm sure it wouldn't be too hard to populate with cases of well-known endocrine hormones acting only in a paracrine fashion.

**Comment by raymond91125** [comment]
Would the merge of the pull request sufficient to close this and help the CL ticket? If so, I'll go ahead.

**Comment by dosumis** [comment]
Hi @raymond91125 Once GO PR is merged with the general term - the next question is, will GO is happy to add endocrine hormone secretion terms for specific endocrine hormones.   IF so, CL can provide a list of the ones we need.   We'd prefer this solution, but if not, just let us know and we'll handle the formalisation on our side.

**Comment by pgaudet** [comment]
I added this to next Monday's ontology editors call agenda.

**Comment by raymond91125** [comment]
> I added this to next Monday's ontology editors call agenda.

Thanks. I'm also not clear on the proper use of GOCHE https://github.com/geneontology/go-ontology/pull/24473.

**Comment by pgaudet** [comment]
Discussion Ontology editors call: 


See: https://en.wikipedia.org/wiki/Endocrine_system 
Events that happen in the organs mentioned in that wiki page  (including [Hypothalamus](https://en.wikipedia.org/wiki/Hypothalamus), [Anterior pituitary gland](https://en.wikipedia.org/wiki/Anterior_pituitary), 
[Pineal gland](https://en.wikipedia.org/wiki/Pineal_gland), [Posterior pituitary gland](https://en.wikipedia.org/wiki/Posterior_pituitary), etc are often endocrine processes, but we think that this is not always the case - for ex insulin can also act on alpha pancreatic cells (to be confirmed) - in this case, it would probably nit be endocrine. 

In short - we dont have an easy way to classify endocrine processes in a way that would be complete and correct (current children of "endocrine process" are very incomplete). Without suggestions to address this, GO editors suggest to obsolete the terms below:   

- [ ] obsolete GO:0060986 endocrine hormone secretion with REPLACE BY GO:0046879 hormone secretion
- [ ] obsolete GO:0050886 endocrine process
- [ ] obsolete  GO:0044060 regulation of endocrine process
- [ ] obsolete GO:0038002 endocrine signaling (no children; 3 EXP annotations)

**Comment by deustp01** [comment]
Whoa!
Go back up into the history of this ticket for discussion of what looks to me like a clear. widely-used distinction among endocrine, paracrine, and autocrine signaling. How can you logically dispose of endocrine as a legitimate category without also disposing of the other two, and do you really want to do that?

**Comment by cmungall** [comment]
I think @deustp01 has a point

**Comment by raymond91125** [comment]
I thought there were decisions made at Ontology Editor mtg (Jan 9th) as @pgaudet laid out.

**Comment by deustp01** [comment]
It looks like there was, and 8 months later I worry that it was wrong as noted in my comment, with apologies for being asleep at the time.

**Comment by raymond91125** [comment]
It appears to me that there is some conflation between endocrine secretion vs GO:0038002 endocrine signaling in this discussion. Whether a hormone is secreted for distant or nearby targets is not always clear cut. Thus it's better to just annotate with hormone secretion and let the signaling term annotation specify the target(s). I propose we only do 
obsolete GO:0060986 endocrine hormone secretion with REPLACE BY GO:0046879 hormone secretion

**Comment by raymond91125** [comment]
@dragon-ai-agent please check if GO:0060986 is used by other ontologies.

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23876635686)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
@raymond91125 Here is what I found regarding external usage of GO:0060986 (endocrine hormone secretion):

**CL (Cell Ontology):** Previously used in the logical definition of CL:0000163 (endocrine cell) as `'secretory cell' and ('capable of' some 'endocrine hormone secretion')`). That reference has since been **removed** from the CL edit file. The original ticket (obophenotype/cell-ontology#1706) was closed and superseded by obophenotype/cell-ontology#1884, which is still open and tracks updating endocrine cell logical definitions, relations guide, and DOSDP patterns.

**UBERON:** Previously used in UBERON:0002368 (endocrine gland) as `SubClassOf 'capable of' some 'endocrine hormone secretion'`. That reference has also been **removed** from the UBERON edit file. The ticket (obophenotype/uberon#2657) was closed in Sept 2022 with no further comments.

**FBbt (Drosophila anatomy):** GO:0060986 is listed in the import terms list (`go_terms.txt` and `non_native_classes.txt`), but it is **not used in any axioms** in the current FBbt edit files. There are references in a tutorial/FUNCARO file defining "endocrine system", "endocrine system component", and "endocrine gland" via `capable_of GO:0060986`, but this is tutorial content, not the live ontology.

**UPHENO:** Has an auto-generated pattern entry mapping GO:0060986 to UPHENO:0052243 (likely "abnormally decreased quality of endocrine hormone secretion"). This is auto-generated from the GO hierarchy and would update if GO:0060986 is obsoleted.

**Summary:** The two primary ontology consumers that previously depended on GO:0060986 (CL and UBERON) have already removed their references. FBbt lists it as an import but does not use it in axioms. Obsoleting GO:0060986 with a REPLACE BY to GO:0046879 (hormone secretion) should be safe from an external ontology perspective, though it would be courteous to notify the FBbt team to clean up their import list.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @raymond91125
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23876635686)

**Comment by raymond91125** [comment]
@dragon-ai-agent please obsolete GO:0060986 endocrine hormone secretion with REPLACE BY GO:0046879 hormone secretion.

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23877109613)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
@raymond91125 Done. PR #31813 obsoletes GO:0060986 (endocrine hormone secretion) with replaced_by GO:0046879 (hormone secretion).

The 7 former child terms of GO:0060986 have been rewired to point directly to GO:0046879 (hormone secretion):

- GO:0032274 gonadotropin secretion
- GO:0032334 inhibin secretion
- GO:0035898 parathyroid hormone secretion
- GO:0035929 steroid hormone secretion
- GO:0045443 juvenile hormone secretion
- GO:0051458 corticotropin secretion
- GO:0070162 adiponectin secretion

No direct annotations exist for GO:0060986, so no annotation migration is needed. CL and UBERON have already removed their references to this term (confirmed in prior research above).

All validation checks pass.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @raymond91125
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23877109613)
